### ISSUE TITLE
CSS code highlighting not being applied

### ISSUE THREAD

**Issue by patmigliaccio** [patmigliaccio/notion-md-pdf#1]
Issue with `markdown-pdf` not applying CSS properly when printing to PDF. Might be a result of a `PhantomJS` issue.

**Comment by patmigliaccio** [comment]
Potentially use [marked](https://github.com/chjj/marked) with [wkhtmltopdf](http://wkhtmltopdf.org/) or [node-wkhtmltopdf](https://github.com/devongovett/node-wkhtmltopdf) as suggested in https://github.com/alanshaw/markdown-pdf/issues/82#issuecomment-182142195

e.g
```javascript
const marked = require('marked'),
    wkhtmltopdf = require('wkhtmltopdf');

let html = marked(mdString);

wkhtmltopdf(html)
    .pipe(fs.createWriteStream('out.pdf'));
```

**Comment by patmigliaccio** [comment]
Closed by: ededb2a94294d81abdd0e0d685c7a17ed06d2090
